changelog/undistributed/changelog_show_security_policies_hit-count-23aug2021.rst

```diff
@@ -0,0 +1,6 @@
+--------------------------------------------------------------------------------
+                            New
+--------------------------------------------------------------------------------
+* JUNOS
+    * Add ShowSecurityPoliciesHitCount parser:
+        * Parse policies per logical-system
```

Comment:
I think this should be:
* Added howSecurityPoliciesHitCount:
    * show security policies hit-count
 
To conform to the template better.